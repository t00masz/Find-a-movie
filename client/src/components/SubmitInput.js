import React from 'react';

const SubmitInput = (props) => (
  <div>
    <input className="button" type='submit' value={props.title}/>
  </div>
);

export default SubmitInput;
