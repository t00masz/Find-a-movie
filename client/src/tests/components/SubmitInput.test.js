import React from 'react';
import { shallow } from 'enzyme'
import SubmitInput from '../../components/SubmitInput';

test('should render submit input', () => {
    const wrapper = shallow(<SubmitInput title="Send data!" />);
    expect(wrapper).toMatchSnapshot();
});
