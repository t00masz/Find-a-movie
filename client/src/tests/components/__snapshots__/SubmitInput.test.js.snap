// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render submit input 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SubmitInput
    title="Send data!"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <input
        className="button"
        type="submit"
        value="Send data!"
/>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "className": "button",
        "type": "submit",
        "value": "Send data!",
      },
      "ref": null,
      "rendered": null,
      "type": "input",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <input
          className="button"
          type="submit"
          value="Send data!"
/>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "button",
          "type": "submit",
          "value": "Send data!",
        },
        "ref": null,
        "rendered": null,
        "type": "input",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
